packages/devtools_app/lib/src/timeline/flutter/flutter_frames_chart.dart

```diff
@@ -2,20 +2,669 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
+import 'dart:async';
+import 'dart:ui';
+
 import 'package:flutter/material.dart';
 
-class FlutterFramesChart extends StatelessWidget {
+import 'package:mp_chart/mp/chart/bar_chart.dart';
+import 'package:mp_chart/mp/controller/bar_chart_controller.dart';
+import 'package:mp_chart/mp/core/adapter_android_mp.dart';
+import 'package:mp_chart/mp/core/axis/x_axis.dart';
+import 'package:mp_chart/mp/core/data/bar_data.dart';
+import 'package:mp_chart/mp/core/data_set/bar_data_set.dart';
+import 'package:mp_chart/mp/core/description.dart';
+import 'package:mp_chart/mp/core/enums/limite_label_postion.dart';
+import 'package:mp_chart/mp/core/enums/x_axis_position.dart';
+import 'package:mp_chart/mp/core/entry/bar_entry.dart';
+import 'package:mp_chart/mp/core/entry/entry.dart';
+import 'package:mp_chart/mp/core/highlight/highlight.dart';
+import 'package:mp_chart/mp/core/limit_line.dart';
+import 'package:mp_chart/mp/core/marker/line_chart_marker.dart';
+import 'package:mp_chart/mp/core/poolable/point.dart';
+import 'package:mp_chart/mp/core/utils/color_utils.dart';
+import 'package:mp_chart/mp/core/utils/painter_utils.dart';
+import 'package:mp_chart/mp/core/value_formatter/default_value_formatter.dart';
+import 'package:mp_chart/mp/core/value_formatter/value_formatter.dart';
+
+class FlutterFramesChart extends StatefulWidget {
+  const FlutterFramesChart();
+
+  @override
+  FlutterFramesChartState createState() => FlutterFramesChartState();
+}
+
+class FlutterFramesChartState extends State<FlutterFramesChart> {
+  BarChartController _chartController;
+
+  BarChartController get chartController => _chartController;
+
+  // Index into the raw data.
+  int timerDataIndex;
+
+  // Active timer running.
+  Timer _timer;
+
+  // Datapoint entry for each UI duration value (Stacked bars).
+  final List<BarEntry> _frameDurations = <BarEntry>[];
+
+  // Trace #1 UI Duration.
+  BarDataSet frameDurationsSet;
+
+  @override
+  void initState() {
+    _initController();
+
+    // TODO(terry): Pass true to simulate live feed.
```

Comment:
remove todo? true is already being passed to `_initData`

Reply:
Should read:

    // True simulates charting a live feed, false to chart all canned data immediately.